vars/stepResult.groovy

```diff
@@ -31,6 +34,7 @@ def call(Map config= [:]) {
   } else {
     param['context'] = config['context'] + "/" + config['name']
   }
+  def flow_name = config.get('flow_name', env.STAGE_NAME)
```

Comment:
```suggestion
```

Reply:
If we leave all of the flow_name parameters in, shouldn't this be:
```suggestion
  def flow_name = config.get('flow_name', context + '/' + env.STAGE_NAME)
```
?

Reply:
Your right, it is inconsistent.  The default construction for the flow_name needs to be the same every where it is calculated.   I missed that edit.

Reply:
But leaving this here as is, means it's not going to find the flow node doesn't it, because it's name is:

```
    "name": "build/Build RPM on CentOS 7",
```

But you are setting it to `env.STAGE_NAME` which will mean for the above case, it would by trying to look for:

```
    "name": "Build RPM on CentOS 7",
```

but we know that isn't going to exist.  It has `build/` prefixed to the `env.STAGE_NAME`.

Reply:
With the last push, the flow_name for buildRpm is now "Build RPM for CentOS 7" set inside of pipeline-lib as the default.
The buildRpmPost step calls stepResult specifying the flow_name that it also defaults to env.STAGE_NAME.

So that worked before with the change, and still works after, as long as buildRpm and buildRpmPost are consistent with each other.

The Jenkinsfiles have places where they call stepResult and need the default for the flow_name to be env.STAGE_NAME.